samples/Blazor.ExampleConsumer/Components/Pages/ReadToMe.razor.cs

```diff
@@ -74,10 +74,6 @@ async Task GetVoicesAsync(bool isFromCallback = false)
 
     void OnTextChanged(ChangeEventArgs args) => _text = args.Value?.ToString();
 
-    void OnVoiceSpeedChange(ChangeEventArgs args) =>
```

Comment:
As a sample I've added the <InvariantGlobalization> tag to the csproj, this permit to use the "@bind" directive in the range slider without the need to parse the double from a string, because at list in italian the decimal separator was causing issues parsing 0.75 into 75

Reply:
Even though I think that "InvariantGlobalization" probably wasn't necessary (?). Maybe the direct "bind" would have solved. I'll try it later